Azure-Samples/modern-data-warehouse-dataops
# notebook-dataframe (non-nutter tests)

# Type of PR
- Code changes
- Test changes
- CI-CD changes

## Purpose
Larger part of Databricks-Ops samples. Adds a sample on how to do automated test on Databricks notebooks with inline assert statements as part of a CI/CD pipeline.

## Does this introduce a breaking change? If yes, details on what can break
No 
## Author pre-publish checklist


## Validation steps
See included README to test out sample. 
Furthermore, see ci and cd pipeline runs (see PR checks).

## Issues Closed or Referenced

- Closes #329
